lib/chewy/railtie.rb

```diff
@@ -1,5 +1,15 @@
 module Chewy
   class Railtie < Rails::Railtie
+    def self.all_engines
+      engines = if Rails.application.railties.respond_to?(:engines)
+        Rails.application.railties.engines
+      else
+        Rails::Engine.subclasses.map(&:instance)
+      end
```

Comment:
Currently this looks great, but I still don't understand, why do we need this check? It seems `Rails::Engine.subclasses.map(&:instance)` supposed to work with any rails >= 3.2